api/lib/redisHelper.mjs

```diff
@@ -26,17 +27,24 @@ export function getClient(databaseIndex = 0) {
  * Gets the value of a specified key in the database.
  * @param {string} key the key of the entry to get.
  * @param {number} databaseIndex the index the entry is stored in.
+ * @throws {NotFoundError} if the key is not in the database.
  * @returns {object} the entry's information.
  */
 export async function getEntry(key, databaseIndex = 0) {
     const client = getClient(databaseIndex);
     await client.connect();
 
-    const res = await client.get(key);
-    // TODO: Throw error if the key does not exist in the db.
-
-    await client.quit();
-
+    let res;
+    try {
+        res = await client.get(key);
```

Comment:
```suggestion
    try {
        const res = await client.get(key);
```

Rule of thumb: use `const` whenever possible in js (only use `let` if you have to mutate the value for some reason.  Basically never use `var`).